### PR TITLE
Feat/tag augury logged error alarms

### PR DESCRIPTION
**Pull request by svevang**
Fixes up the previous #669 

This:

- adds the tags to the "logged errors" alarms
- reformats so the metric filters, alarms and tags are cuddled together

### CODE REVIEW

File: stacks/apps/augury.yml [hunk 1]
```diff
@@ -738,6 +744,32 @@ Resources:
       Statistic: Sum
       Threshold: 0
       TreatMissingData: notBreaching
+  AuguryWorkerLoggedAlarmTags:
```
Comment (farski) [inline stacks/apps/augury.yml:747]: `AuguryWorkerLoggedErrorsAlarmTags`

File: stacks/apps/augury.yml [hunk 2]
```diff
@@ -738,6 +744,32 @@ Resources:
...
+    Type: Custom::CloudWatchAlarmTags
+    Condition: EnableWorkers # See README
```
Comment (farski) [inline stacks/apps/augury.yml:749]: No condition on alarm

File: stacks/apps/augury.yml [hunk 3]
```diff
@@ -752,6 +784,22 @@ Resources:
       Statistic: Sum
       Threshold: 0
       TreatMissingData: notBreaching
+  AugurySlowWorkerLoggedAlarmTags:
```
Comment (farski) [inline stacks/apps/augury.yml:787]: `AugurySlowWorkerLoggedErrorsAlarmTags`

File: stacks/apps/augury.yml [hunk 4]
```diff
@@ -752,6 +784,22 @@ Resources:
...
+    Type: Custom::CloudWatchAlarmTags
+    Condition: EnableWorkers # See README
```
Comment (farski) [inline stacks/apps/augury.yml:789]: No condition on alarm

**Review by farski — CHANGES_REQUESTED** [review]

File: stacks/apps/augury.yml [hunk 5]
```diff
@@ -724,6 +704,32 @@ Resources:
       Statistic: Sum
       Threshold: 0
       TreatMissingData: notBreaching
+  AuguryWebLoggedAlarmTags:
```
[the remark this answers is not in the document]
Comment (svevang) [reply stacks/apps/augury.yml]: Thank you for the review!   I looked at the other names and opted to drop "Augury" from these resources names.  It already is in the augury stack etc, so no need for the appname prefix.  And less typing to boot!

**Comment by svevang** [comment]
In the end I'd like the slack alarms to link to the log files.  I think this PR gets us a bit closer.

It seems to get the reference to the log lines, I'd have to add one of these: https://github.com/PRX/Infrastructure/blob/ed1556a0b7cc6a64eb8ec4a372f2cf43f981306f/devops/tooling/chat-ops/cloudwatch-alarms/src/slack/log-groups.js#L11
 to the alarm resources.    Maybe?

**Comment by farski** [comment]
Yes, as long as the namespace that an alarm belongs to is in that list and the alarm includes the `prx:ops:cloudwatch-log-group-name` tag with an appropriate value, the Slack messages will include a link to CloudWatch Logs Insights for the period the alarm covers. If you need to add any namespaces to that list, I wouldn't worry about a PR; just add them.

File: stacks/apps/augury.yml [hunk 6]
```diff
@@ -752,6 +782,21 @@ Resources:
       Statistic: Sum
       Threshold: 0
       TreatMissingData: notBreaching
+  SlowWorkerLoggedAlarmTags:
```
Comment (farski) [inline stacks/apps/augury.yml:785]: `SlowWorkerLoggedErrorsAlarmTags`

**Review by farski — CHANGES_REQUESTED** [review]

**Review by farski — APPROVED** [review]

**Pull request merged**